extendr-api/Cargo.toml

```diff
@@ -15,12 +15,12 @@ license = "MIT"
 repository = "https://github.com/extendr/extendr"
 
 [dependencies]
-libR-sys = "0.3.0"
-extendr-macros = { path = "../extendr-macros", version = "0.3.1" }
-extendr-engine = { path = "../extendr-engine", version = "0.3.1" }
-ndarray = { version = "0.15.3", optional = true }
+libR-sys = { workspace = true }
+extendr-macros = { path = "../extendr-macros" }
+extendr-engine = { path = "../extendr-engine" }
 lazy_static = "1.4"
 paste = "1.0.5"
+ndarray = { version = "0.15.3", optional = true }
```

Comment:
Why an ndarray change?

Reply:
Accidental. I think I've cleaned it up now.